tests/integration/cli/classification/test_classification.py

```diff
@@ -102,6 +102,14 @@ def test_otx_train(self, template, tmp_dir_path):
         args1["--load-weights"] = f"{template_work_dir}/trained_{template.model_template_id}/weights.pth"
         otx_train_testing(template, tmp_dir_path, otx_dir, args1)
 
+    @e2e_pytest_component
```

Comment:
I think it's better to test this before test_otx_train because below eval, export test expect the weight comes from normal incremental learning

Reply:
Ok, got it.